packages/ComponentLibrary/src/components/Table/sections/CenterSection.tsx

```diff
@@ -0,0 +1,53 @@
+import { Box, useTheme } from '@mui/material';
```

Comment:
This does not seem to be a component, in any case make a base "Section" component and move the component to MainUI.